packages/demo/usecase-demo/src/components/CarouselScrollArrows.tsx

```diff
@@ -1,11 +1,23 @@
-import { RefObject } from 'react';
+import { RefObject, useEffect, useState } from 'react';
 import { ArrowLeft, ArrowRight } from 'react-feather';
 
 export function CarouselScrollArrows({
   carousel,
 }: {
   carousel: RefObject<HTMLDivElement>;
 }) {
+  const [showArrows, setShowArrows] = useState<boolean>();
+  useEffect(() => {
+    const width = carousel.current?.getBoundingClientRect().width;
+    const nbChild = carousel.current?.childElementCount;
+    const childrenWidth = nbChild * (400 + 16) - 16;
```

Comment:
Wow, I kinda understand but not sure. 400px is a card's width? Let's have a constant somewhere:
```
const ONE_CARD_WIDTH = 400;
```

Also Content cards seem to be 400px but Creator cards are 250px 😬
(Route http://localhost:5173/subscribe)